### DISCUSSION TITLE
Reference not picking up the value

### DISCUSSION THREAD

**Discussion by alixain** [firecmsco/firecms#233] [Q&A] [answered]
<img width="448" alt="Screenshot 2022-02-17 at 4 10 21 PM" src="https://user-images.githubusercontent.com/14543921/154469564-a5aa70f2-cec7-43b5-8c7d-55b6aaf55c1b.png">

<img width="303" alt="Screenshot 2022-02-17 at 4 10 49 PM" src="https://user-images.githubusercontent.com/14543921/154469618-4865e9cc-8bf6-4fa9-bccc-e5bb9b5d346e.png">

<img width="1004" alt="Screenshot 2022-02-17 at 4 11 43 PM" src="https://user-images.githubusercontent.com/14543921/154469800-c26e7a34-d359-46db-b0b3-abc5941eee53.png">

_Originally posted by @alixain in https://github.com/Camberi/firecms/issues/183#issuecomment-1042838446_

**Comment by alixain** [comment]
<img width="334" alt="Screenshot 2022-02-17 at 4 21 03 PM" src="https://user-images.githubusercontent.com/14543921/154471250-6e5fcebb-6101-4078-8bd8-876bca30f09f.png">

<img width="278" alt="Screenshot 2022-02-17 at 4 21 11 PM" src="https://user-images.githubusercontent.com/14543921/154471284-241277f6-5191-4bec-8769-88c2082fa19d.png">


All references are correct

**Comment by fgatti675** [comment, marked as answer]
Hi @alixain 
I moved this to Q&A since it is more appropriate.
The `brand` field seems to be saved as a string and not a reference.
FireCMS currently does not support that
